### PR TITLE
0x33 caller

### PR DESCRIPTION
**Pull request by danilowhk**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- 0x33 Caller Opcode implementation
-
-

### CODE REVIEW

File: src/kakarot/instructions/environmental_information.cairo [hunk 1]
```diff
@@ -45,4 +47,29 @@ namespace EnvironmentalInformation {
...
+    // @dev Get caller address.
+    // @custom:since Frontier
+    // @custom:group Environmental Information
+    // @custom:gas 3
+    // @custom:stack_consumed_elements 0
+    // @custom:stack_produced_elements 1
+    // @return The pointer to the updated execution context.
+    func exec_caller{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+        ctx: model.ExecutionContext*
+    ) -> model.ExecutionContext* {
```
Comment (abdelhamidbakhta) [inline src/kakarot/instructions/environmental_information.cairo:61]: Small typo here: CALL should be CALLER

**Review by abdelhamidbakhta — APPROVED** [review]
One small typo, otherwise looks good

File: src/kakarot/instructions/environmental_information.cairo (hunk 1 above)
Comment (danilowhk) [reply src/kakarot/instructions/environmental_information.cairo:61]: Typo fixed and conflicts resolved :)

**Review by abdelhamidbakhta — APPROVED** [review]
Lgtm

**Pull request closed without merging**